stable/airflow/values.yaml

```diff
@@ -137,6 +137,22 @@ airflow:
   podAnnotations: {}
     ## Example:
     ## iam.amazonaws.com/role: airflow-Role
+  extraContainers: []
+  ## Additional containers to run alongside the Scheduler, Worker and Web pods
+  ## This could, for example, be used to run a sidecar that syncs DAGs from object storage.
```

Comment:
interesting, if you have a working solution, could you (later) describe it in the README?

Reply:
Yes, sure!